src/sgame/sg_main.cpp

```diff
@@ -2359,6 +2363,18 @@ void G_RunFrame( int levelTime )
 	// now we are done spawning
 	level.spawning = false;
 
+	// cancel vote if timed out
+	for ( i = 0; i < NUM_TEAMS; i++ )
+	{
+		G_CheckVote( (team_t) i );
+	}
```

Comment:
where is this coming from?

Reply:
Ooops I forgot to remove it below...Nice catch.